app/src/main/java/com/example/fragmentassg2/FragmentLogin.kt

```diff
@@ -19,49 +21,42 @@ class FragmentLogin : Fragment() {
         return inflater.inflate(R.layout.login_fragment,container,false)
     }
 
-
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        btnHindi?.setOnClickListener(this)
+        btnSkip?.setOnClickListener(this)
+        btnLogin?.setOnClickListener(this)
+    }
 
-        tvGood.setOnClickListener(){
-            Toast.makeText(context,"Good To See You Again",Toast.LENGTH_SHORT).show()
-        }
-
-        tvDownload.setOnClickListener(){
-            Toast.makeText(context,"Download here",Toast.LENGTH_SHORT).show()
-        }
+    override fun onClick(v: View?) {
+        when(v){
 
-        tvForgot.setOnClickListener(){
-            Toast.makeText(context,"Forgot PassWord",Toast.LENGTH_SHORT).show()
-        }
+            btnHindi->Toast.makeText(context,getString(R.string.Hindi),Toast.LENGTH_SHORT).show()
+            btnSkip->Toast.makeText(context,getString(R.string.skip),Toast.LENGTH_SHORT).show()
 
-        tvSSO.setOnClickListener(){
-            Toast.makeText(context,"Login with SSO",Toast.LENGTH_SHORT).show()
-        }
+            btnLogin->{
+                val number ="9953755955"
 
-        tvSign.setOnClickListener(){
-            Toast.makeText(context,"Sign up",Toast.LENGTH_SHORT).show()
-        }
+                if(editPhone.editText?.text.toString()==number){
+                    val fragment=FragmentPassword()
```

Comment:
@raghav32 make this global, we should not create any variables inside onClick listener